examples/react-native/package.json

```diff
@@ -24,20 +28,31 @@
     "@babel/core": "^7.17.10",
     "@babel/runtime": "^7.12.5",
     "@react-native-community/eslint-config": "^2.0.0",
+    "@storybook/addon-actions": "^5.3",
+    "@storybook/addon-knobs": "^5.3",
+    "@storybook/addon-links": "^5.3",
+    "@storybook/addon-ondevice-actions": "^5.3.23",
+    "@storybook/addon-ondevice-knobs": "^5.3.26",
+    "@storybook/react-native": "^5.3.25",
+    "@storybook/react-native-server": "^5.3.23",
     "@types/jest": "^26.0.23",
     "@types/react-native": "^0.67.3",
     "@types/react-test-renderer": "^17.0.1",
     "@typescript-eslint/eslint-plugin": "^5.17.0",
     "@typescript-eslint/parser": "^5.17.0",
     "babel-jest": "^26.6.3",
+    "babel-loader": "^8.2.5",
     "eslint": "^7.32.0",
     "jest": "^26.6.3",
     "metro-react-native-babel-preset": "^0.70.2",
+    "react-dom": "17.0.2",
+    "react-native-storybook-loader": "^2.0.5",
     "react-test-renderer": "^17.0.2",
     "typescript": "^4.6.3"
   },
   "resolutions": {
-    "@types/react": "^17"
+    "@types/react": "^17",
+    "**/@types/react": "^17"
```

Comment:
To my knowledge, resolutions don't work unless it's in the monorepo root `package.json`: https://github.com/yarnpkg/yarn/issues/5039. 

Have you found this to work?

Reply:
Yes..it works :)  and if you read the issue carefully, towards the end there's a post saying "Still happens to me with the version 1.21.1 and looks to be fixed with the last version 1.22.4". We use 1.2219.

Reply:
Cool, thanks for confirming! We'll just have to make sure we enforce minimum `yarn` across the team then.